packages/morphomax-frontend/vite.config.ts

```diff
@@ -4,33 +4,42 @@ import tailwindcss from '@tailwindcss/vite';
 import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
 
-export default defineConfig({
-  plugins: [
-    react(),
-    tailwindcss(),
-    sentryVitePlugin({
-      org: 'lit-protocol-lw',
-      project: 'vincent-yield-frontend',
-    }),
-    sentryVitePlugin({
-      org: 'lit-protocol-lw',
-      project: 'vincent-yield-frontend',
-    }),
-  ],
+export default defineConfig(({ command }) => {
+  const isBuild = command === 'build';
 
-  define: {
-    global: 'globalThis',
-  },
-  optimizeDeps: {
-    include: ['@lit-protocol/vincent-app-sdk/jwt', '@lit-protocol/vincent-app-sdk/webAuthClient'], // Dev: delete node_modules/.vite when rebuilding this
-  },
-  resolve: {
-    alias: {
-      '@': path.resolve(__dirname, './src'),
-      buffer: 'buffer/',
+  const sentryEnabled =
+    isBuild &&
+    ((process.env.VERCEL === '1' && process.env.VERCEL_ENV === 'production') ||
+      String(process.env.SENTRY_UPLOAD_SOURCEMAPS).toLowerCase() === 'true');
+
+  return {
+    plugins: [
+      react(),
+      tailwindcss(),
+      ...(sentryEnabled
+        ? [
+            sentryVitePlugin({
+              org: 'lit-protocol-lw',
+              project: 'vincent-yield-frontend',
+              authToken: process.env.SENTRY_AUTH_TOKEN,
+            }),
+          ]
+        : []),
+    ],
+    define: {
+      global: 'globalThis',
+    },
+    optimizeDeps: {
+      include: ['@lit-protocol/vincent-app-sdk/jwt', '@lit-protocol/vincent-app-sdk/webAuthClient'], // Dev: delete node_modules/.vite when rebuilding this
```

Comment:
Just curious why we needed to add these to optimizeDeps -- is there something wrong with our NPM packages that forced you to use this flag to work around an issue with how we're publishing them?  I don't want consumers of the app-sdk to need to learn about this just to use our package if we can avoid it 🤔

Reply:
IIRC it was because of having them linked locally. Unless it is us, they wouldn't need to do this